source/user/gto.rst

```diff
@@ -233,10 +232,77 @@ but the ``STO-3G`` basis will be used for the atom ``H2``:
 
 See :source:`examples/gto/04-input_basis.py` for more examples.
 
+Ordering of basis functions
+---------------------------
+GTO basis functions are sorted in the following order: (1) atoms, (2) angular
+momentum, (3) shells, (4) spherical harmonics. Particularly, basis functions are
+grouped in terms of the atoms they are assigned to. On each atom, basis
+functions are grouped according to their angular momentum. In each group of
+angular momentum, the ordering index is shell. A shell can be the atomic shell
+or just the group of primitive Gaussian functions with the same exponent. Shells
+are sorted from inner shell to outer shell (or from large exponents to small
+exponents). In each shell, the spherical parts of the GTO basis follow the
+Condon-Shortley convention, with the ordering (and phase) the same to
+https://en.wikipedia.org/wiki/Table_of_spherical_harmonics
+except `p` functions. The p functions are sorted in the order of `px`, `py`,
+`pz` instead of the ordering `py`, `pz`, `px` in the table above.
```

Comment:
```suggestion
Condon-Shortley convention, with the ordering (and phase) given in 
`the Wikipedia table of spherical harmonics <https://en.wikipedia.org/wiki/Table_of_spherical_harmonics>`, 
except for the `p` functions for which the order of `px`, `py`, `pz` is used 
instead of the order`py`, `pz`, `px` used in the table above.
```